frontend/src/__tests__/cypress/cypress/support/commands/odh.ts

```diff
@@ -339,14 +352,149 @@ declare global {
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListPipelineRunsResourceKF | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineRunKFv2 | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `DELETE /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<PipelineRunKFv2 | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListExperimentsResponseKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments/:experimentId`,
+        options: { path: { namespace: string; experimentId: string } },
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments/:experimentId`,
+        options: { path: { namespace: string; experimentId: string } },
+        response: OdhResponse<ExperimentKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/recurringruns`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineRunJobKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/recurringruns`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListPipelineRunJobsResourceKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: 'GET /api/service/modelregistry/modelregistry-sample/api/model_registry/v1alpha3/registered_models/1/versions',
+        response: OdhResponse<ModelVersionList | undefined>,
```

Comment:
The frontend API in `modelRegistry/custom.ts` shows that this API only returns `ModelVersionList`.
```suggestion
        response: OdhResponse<ModelVersionList>,
```